src/app/view/perspectives/post-it/document/post-it-document.component.ts

```diff
@@ -44,6 +44,14 @@ import Update = DocumentsAction.Update;
 })
 export class PostItDocumentComponent implements OnInit, AfterViewInit, OnDestroy {
 
+  @HostListener('focusout')
+  public onFocusOut(): void {
+    if (this.changed) {
+      this.changed = false;
+      this.changes.emit();
```

Comment:
It seems that you have changed the way post-its are saved. However, it didn't solve the refresh problem I previously mentioned. Wasn't this change supposed to fix it? Or what was the reason behind changing this?

Reply:
This is a pretty significant optimalization. If you chcecked in the previous version how often it sends data without needing to.